engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/Name.scala

```diff
@@ -206,7 +205,7 @@ object Name {
     * @param diagnostics compiler diagnostics for this node
     * @return a copy of `this`, updated with the specified values
     */
-  sealed case class Qualified(
+  final case class Qualified(
```

Comment:
What is it, seal-hunting season? :seal:

Reply:
`sealed` doesn't really guarantee there is no subclass (I think). With `final` I can be sure there is no subclass.